packages/sources/layer2-sequencer-health/src/config/index.ts

```diff
@@ -1,8 +1,11 @@
 import { AdapterConfigError, Requester, util } from '@chainlink/ea-bootstrap'
 import { AdapterContext, Config } from '@chainlink/ea-bootstrap'
 import { envDefaultOverrides } from './envDefaultOverrides'
+import { SequencerProvider } from 'starknet'
 
 export const NAME = 'L2_SEQUENCER_HEALTH'
+export const DEFAULT_PRIVATE_KEY =
```

Comment:
This was just moved from a different file.  It was previously used to instantiate a dummy wallet to make dummy transactions to EVM chains.